src/Ng/ElfieFromCatalogCollector.cs

```diff
@@ -64,15 +65,43 @@ public ElfieFromCatalogCollector(Version indexerVersion, Version mergerVersion,
                 // Load the download counts  
                 JArray downloadJson = FetchDownloadCounts(this._downloadCountsUri);
 
-                // Get the packages to include in the ardb index
-                IEnumerable<RegistrationIndexPackage> sortedPackagesToInclude = GetPackagesToInclude(downloadJson, this._downloadPercentage);
+                // We need to get the list of packages from two different sources. The first source is NuGet
```

Comment:
It would have been cleaner if I had created a NuGet provider type. That way we could have had two providers, one for NuGet and one for local packages. It would have made this file smaller and easier to read. ... it just never occurred to me that we might have more than one provider for NuGet packages. ... and I didn't want to make that change now because it seemed too risky.